doc/1/getting-started/kotlin/index.md

```diff
@@ -0,0 +1,126 @@
+
+::: info
+The following examples are made to be executed without any IDE.
+If you're using Eclipse, IntelliJ or another Java IDE, you need to add the SDK as a project dependency in your classpath.
+:::
+
+### Installing on Android using gradle
+
+In your app build.gradle add the following line and sync.
+
+    implementation "io.kuzzle:kuzzle-sdk-jvm:1.0.0"
+
+## First connection
+
+Initialize a new Java project, create a `gettingstartedfirstconnection.kt` file and start by adding the code below:
+
+<<< ./snippets/firstconnection.kt
+
+This program initializes the Kuzzle Server storage by creating a index, and a collection inside it
+Run the program with the following command:
+
+```bash
+$ javac -classpath ./path/to/the/sdk.jar gettingstartedfirstconnection.java
+$ java -classpath .:./path/to/the/sdk.jar gettingstartedfirstconnection
+Connected!
+Index nyc-open-data created!
+Collection yellow-taxi created!
+```
+
+Congratulations, you performed your first connection to Kuzzle Server via a Java program.
+You now know how to:
+
+- Instantiate Kuzzle SDK and connect to Kuzzle Server using a specific protocol (here `websocket`)
+- Create a index
+- Create a collection within an existing index
+
+## Create your first document
+
+Now that you successfully connected to your Kuzzle Server instance, and created an index and a collection, it's time to manipulate some data.
+
+Here is how Kuzzle structures its storage space:
+
+- indexes contain collections
+- collections contain documents
+  Create a `gettingstartedstorage.kt` file in the playground and add this code:
+
+<<< ./snippets/document.kt
+
+As you did before, build and run your program:
+
+```bash
+$ javac -classpath ./path/to/the/sdk.jar  gettingstartedstorage.java
+$ java -classpath .:./path/to/the/sdk.jar gettingstartedstorage
+Connected!
+New document added to yellow-taxi collection!
+```
+
+You can perform other actions such as [delete](/sdk/jvm/1/controllers/document/delete), [replace](/sdk/jvm/1/controllers/document/replace) or [search](/sdk/jvm/1/controllers/document/search) documents. There are also other ways to interact with Kuzzle like our [Admin Console](/core/2/guides/essentials/admin-console), the [Kuzzle HTTP API](/core/2/api/essentials/connecting-to-kuzzle) or by using your [own protocol](/core/2/protocols/essentials/getting-started).
+
+Now you know how to:
+
+- Store documents in a Kuzzle Server, and access those
+
+## Subscribe to realtime document notifications (pub/sub)
+
+Time to use realtime with Kuzzle. Create a new file `gettingstartedrealtime.kt` with the following code:
+
+<<< ./snippets/realtime.kt
+
+This program subscribes to changes made to documents with a `license` field set to `B`, within the `yellow-taxi` collection. Whenever a document matching the provided filters changes, a new notification is received from Kuzzle.
+
+Build and run your program:
+
+```bash
+$ javac -classpath ./path/to/the/sdk.jar gettingstartedrealtime.java
```

Comment:
Files in Kotlin should start with a capital and be CamelCase and shouldn'th this be `gettingstartedrealtime.kt`?